app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateAction.kt

```diff
@@ -23,4 +23,5 @@ sealed interface SwapStateAction {
 
     object SwitchTokens : SwapStateAction
     object SwapSuccess : SwapStateAction
+    object CancelJobs : SwapStateAction
```

Comment:
it is not a feature Action name
we really don't have any jobs in swap context, maybe CancelSwap / CancelSwapLoading?